scripts/linting.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+set -euo pipefail
+
+trap "exit -1" ERR
+
+# Track whether linting failed; we don't want to bail on lint failures
+failed=0
+
+## Main ----------------------------------------------------------------------
+echo "Running Basic Ansible Lint Check"
+
+
+# Install the development requirements.
+if [[ -f "os-ansible-deployment/dev-requirements.txt" ]]; then
+  pip2 install -r os-ansible-deployment/dev-requirements.txt || pip install -r os-ansible-deployment/dev-requirements.txt
```

Comment:
While I agree, I'm also trying to make sure we're not *too* beholden to Travis itself.

If it's a separate dependencies script, that's probably a decent compromise.

Also worth nothing this is taken from upstream OSAD, so this can be improved there, too.